Spikes/movement/Character.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifndef CHARACTER_H
+#define CHARACTER_H
+
+#include "SFML/Graphics.hpp" 
+#include "SFML/OpenGL.hpp" 
+
+using namespace sf;
+
+
+
+class Character
+{
+private:
+	//Character
+	bool m_move;
+	Vector2f m_centre;
+	Vector2f m_pos;
+	float m_radius;
+	float m_speed;
+	float m_angle;
+	CircleShape m_shape;
+
+	//Target
+	Vector2f m_targetPos; 
+	float m_targetRadius;
+	float m_targetAngle;
+	CircleShape m_target;
+
+public:
+	Character();
+	Character(float, float, Color, Color, Vector2f);
+	void update();
+	void draw(RenderWindow&);
+	void resetAngles();
+	float findAngleBetween();
+
+	// Get methods
```

Comment:
should leave short descriptions of methods here in header above definition if not apparent, i.e. resetAngles is obvious now, but later in development it wont be, so a quick comment, update and draw are irrelevant. 

Also findAngleBetween() is an inbuilt method of Aarons Vector2D class, if we can use code thats already made do, i'm aware you probably haven't been told about the Vector2D until now so this is me notifying you. Vector2D can be found on many of the other branches and they should all be the same.